SHARE-256 Refactoring - Adminarea - Tools | Logs

mahmoudZkhalil opened the pull request:
separate functionality in methods
add setter and getter
write phpUnit tests
create invoke method for testing private and protected methods

---
### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroweb-Symfony/blob/develop/.github/contributing.md) and [wiki pages](https://github.com/Catrobat/catroweb-Symfony/wiki/) of this repository.

- [x] Include the name and id of the Jira ticket in the PR’s title eg.: `SHARE-666 The devils ticket`
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no warnings and errors 
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Verify that all tests are passing (CI), if not please state the test cases in the [section](#Tests) below
- [x] Perform a self-review of the changes
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [ ] Verify that your changes do not have any conflicts with the base branch
- [x] Put your ticket into the `Code Review` section in [Jira](https://jira.catrob.at/)
- [ ] Post a message in the *#catroweb* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer
- [ ] Check that your pull request has been successfully deployed to https://web-test-1.catrob.at/

### Additional Description
`TODO: Add additional information that is not in your commit-message here`

### Tests - additional information
`TODO: add additional information about testruns here`

Commit:
SHARE-352 API - PUT /user

-Implement API - PUT /user
-Write tests
-Update DELETE /user tests

Commit:
Changed Media Library API to be consistent with MediaPackageFile Entity

mahmoudZkhalil committed:
SHARE-256 Refactoring - Adminarea - Tools | Logs

separate functionality in methods
add setter and getter
write phpUnit tests
create invoke method for testing private and protected methods

mahmoudZkhalil committed:
SHARE-256 Refactoring - Adminarea - Tools | Logs

separate functionality in methods
add setter and getter
write phpUnit tests
create invoke method for testing private and protected methods

HCrane committed:
SHARE-413

dependabot[bot] committed:
Bump codercat/jwk-to-pem from 0.0.3 to 1.0

Bumps [codercat/jwk-to-pem](https://github.com/acodercat/php-jwk-to-pem) from 0.0.3 to 1.0.
- [Release notes](https://github.com/acodercat/php-jwk-to-pem/releases)
- [Commits](https://github.com/acodercat/php-jwk-to-pem/compare/0.0.3...1.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump standard from 16.0.2 to 16.0.3

Bumps [standard](https://github.com/standard/standard) from 16.0.2 to 16.0.3.
- [Release notes](https://github.com/standard/standard/releases)
- [Changelog](https://github.com/standard/standard/blob/master/CHANGELOG.md)
- [Commits](https://github.com/standard/standard/compare/v16.0.2...v16.0.3)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump google/apiclient from 2.8.2 to 2.8.3

Bumps [google/apiclient](https://github.com/googleapis/google-api-php-client) from 2.8.2 to 2.8.3.
- [Release notes](https://github.com/googleapis/google-api-php-client/releases)
- [Commits](https://github.com/googleapis/google-api-php-client/compare/v2.8.2...v2.8.3)

Signed-off-by: dependabot[bot] <support@github.com>

dmetzner committed:
Merge pull request #1098 from Catrobat/dependabot/npm_and_yarn/standard-16.0.3

Bump standard from 16.0.2 to 16.0.3

dmetzner committed:
Merge pull request #1099 from Catrobat/dependabot/composer/google/apiclient-2.8.3

Bump google/apiclient from 2.8.2 to 2.8.3

dmetzner committed:
Merge pull request #1084 from Catrobat/dependabot/composer/codercat/jwk-to-pem-1.0

Bump codercat/jwk-to-pem from 0.0.3 to 1.0

dependabot[bot] committed:
Bump stylelint from 13.7.2 to 13.8.0

Bumps [stylelint](https://github.com/stylelint/stylelint) from 13.7.2 to 13.8.0.
- [Release notes](https://github.com/stylelint/stylelint/releases)
- [Changelog](https://github.com/stylelint/stylelint/blob/master/CHANGELOG.md)
- [Commits](https://github.com/stylelint/stylelint/compare/13.7.2...13.8.0)

Signed-off-by: dependabot[bot] <support@github.com>

mahmoudZkhalil committed:
SHARE-256

cs fix

mahmoudZkhalil committed:
SHARE-256

cs fix

mahmoudZkhalil committed:
SHARE-256

cs fix and phpunit test fix

mahmoudZkhalil committed:
SHARE-256

cs fix and phpunit test fix

danielzhang130 committed:
Grunt watch needs to copy over custom and analytics js
files before processing

Previous version did not update js files other than
in base directory.

dmetzner committed:
Merge pull request #1100 from Catrobat/dependabot/npm_and_yarn/stylelint-13.8.0

Bump stylelint from 13.7.2 to 13.8.0

mahmoudZkhalil committed:
SHARE-256
phpunit correction to fix phpstan

mahmoudZkhalil committed:
SHARE-256
phpunit correction to fix phpstan

HCrane committed:
Merge pull request #1102 from danielzhang130/grunt_watch_js

Fix Grunt watch not working for custom js files

dependabot[bot] committed:
Bump eslint from 7.13.0 to 7.14.0

Bumps [eslint](https://github.com/eslint/eslint) from 7.13.0 to 7.14.0.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/master/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v7.13.0...v7.14.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump npm from 6.14.8 to 6.14.9

Bumps [npm](https://github.com/npm/cli) from 6.14.8 to 6.14.9.
- [Release notes](https://github.com/npm/cli/releases)
- [Changelog](https://github.com/npm/cli/blob/v6.14.9/CHANGELOG.md)
- [Commits](https://github.com/npm/cli/compare/v6.14.8...v6.14.9)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump eslint-plugin-standard from 4.1.0 to 5.0.0

Bumps [eslint-plugin-standard](https://github.com/standard/eslint-plugin-standard) from 4.1.0 to 5.0.0.
- [Release notes](https://github.com/standard/eslint-plugin-standard/releases)
- [Commits](https://github.com/standard/eslint-plugin-standard/compare/v4.1.0...v5.0.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump friendsofsymfony/jsrouting-bundle from 2.6.0 to 2.7.0

Bumps [friendsofsymfony/jsrouting-bundle](https://github.com/FriendsOfSymfony/FOSJsRoutingBundle) from 2.6.0 to 2.7.0.
- [Release notes](https://github.com/FriendsOfSymfony/FOSJsRoutingBundle/releases)
- [Changelog](https://github.com/FriendsOfSymfony/FOSJsRoutingBundle/blob/master/CHANGELOG.md)
- [Commits](https://github.com/FriendsOfSymfony/FOSJsRoutingBundle/compare/2.6.0...2.7.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump phpstan/phpstan from 0.12.56 to 0.12.57

Bumps [phpstan/phpstan](https://github.com/phpstan/phpstan) from 0.12.56 to 0.12.57.
- [Release notes](https://github.com/phpstan/phpstan/releases)
- [Commits](https://github.com/phpstan/phpstan/compare/0.12.56...0.12.57)

Signed-off-by: dependabot[bot] <support@github.com>

dmetzner committed:
Merge pull request #1106 from Catrobat/dependabot/npm_and_yarn/eslint-plugin-standard-5.0.0

Bump eslint-plugin-standard from 4.1.0 to 5.0.0

dmetzner committed:
Merge pull request #1105 from Catrobat/dependabot/npm_and_yarn/npm-6.14.9

Bump npm from 6.14.8 to 6.14.9

dmetzner committed:
Merge pull request #1108 from Catrobat/dependabot/composer/phpstan/phpstan-0.12.57

Bump phpstan/phpstan from 0.12.56 to 0.12.57

dmetzner committed:
Merge pull request #1107 from Catrobat/dependabot/composer/friendsofsymfony/jsrouting-bundle-2.7.0

Bump friendsofsymfony/jsrouting-bundle from 2.6.0 to 2.7.0

dmetzner committed:
Merge pull request #1104 from Catrobat/dependabot/npm_and_yarn/eslint-7.14.0

Bump eslint from 7.13.0 to 7.14.0

dmetzner reviewed (changes requested):
The refactoring looks good to me. Just one minor thing. Please squash your commits.
Having only 1 commit instead of 4 makes the reviewing process a lot easier.

dependabot[bot] committed:
Bump vimeo/psalm from 4.1.1 to 4.2.1

Bumps [vimeo/psalm](https://github.com/vimeo/psalm) from 4.1.1 to 4.2.1.
- [Release notes](https://github.com/vimeo/psalm/releases)
- [Commits](https://github.com/vimeo/psalm/compare/4.1.1...4.2.1)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump lexik/jwt-authentication-bundle from 2.9.0 to 2.10.0

Bumps [lexik/jwt-authentication-bundle](https://github.com/lexik/LexikJWTAuthenticationBundle) from 2.9.0 to 2.10.0.
- [Release notes](https://github.com/lexik/LexikJWTAuthenticationBundle/releases)
- [Changelog](https://github.com/lexik/LexikJWTAuthenticationBundle/blob/master/CHANGELOG.md)
- [Commits](https://github.com/lexik/LexikJWTAuthenticationBundle/compare/v2.9.0...v2.10.0)

Signed-off-by: dependabot[bot] <support@github.com>

dmetzner committed:
Merge pull request #1110 from Catrobat/dependabot/composer/lexik/jwt-authentication-bundle-2.10.0

Bump lexik/jwt-authentication-bundle from 2.9.0 to 2.10.0

crowdin-bot committed:
New Crowdin translations by Github Action

dependabot[bot] committed:
Bump sweetalert2 from 10.10.1 to 10.10.2

Bumps [sweetalert2](https://github.com/sweetalert2/sweetalert2) from 10.10.1 to 10.10.2.
- [Release notes](https://github.com/sweetalert2/sweetalert2/releases)
- [Changelog](https://github.com/sweetalert2/sweetalert2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sweetalert2/sweetalert2/compare/v10.10.1...v10.10.2)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump phpstan/phpstan-doctrine from 0.12.22 to 0.12.23

Bumps [phpstan/phpstan-doctrine](https://github.com/phpstan/phpstan-doctrine) from 0.12.22 to 0.12.23.
- [Release notes](https://github.com/phpstan/phpstan-doctrine/releases)
- [Commits](https://github.com/phpstan/phpstan-doctrine/compare/0.12.22...0.12.23)

Signed-off-by: dependabot[bot] <support@github.com>

HCrane committed:
Merge pull request #1111 from Catrobat/dependabot/npm_and_yarn/sweetalert2-10.10.2

Bump sweetalert2 from 10.10.1 to 10.10.2

HCrane committed:
Merge pull request #1112 from Catrobat/dependabot/composer/phpstan/phpstan-doctrine-0.12.23

Bump phpstan/phpstan-doctrine from 0.12.22 to 0.12.23

HCrane committed:
Merge pull request #1103 from Catrobat/l10n_crowdin_action

New Crowdin translations by Github Action

dependabot[bot] committed:
Bump phpstan/phpstan-doctrine from 0.12.23 to 0.12.24

Bumps [phpstan/phpstan-doctrine](https://github.com/phpstan/phpstan-doctrine) from 0.12.23 to 0.12.24.
- [Release notes](https://github.com/phpstan/phpstan-doctrine/releases)
- [Commits](https://github.com/phpstan/phpstan-doctrine/compare/0.12.23...0.12.24)

Signed-off-by: dependabot[bot] <support@github.com>

HCrane committed:
Merge pull request #1114 from Catrobat/dependabot/composer/phpstan/phpstan-doctrine-0.12.24

Bump phpstan/phpstan-doctrine from 0.12.23 to 0.12.24

HCrane committed:
Manual Fixes

dependabot[bot] committed:
Bump core-js from 3.7.0 to 3.8.0

Bumps [core-js](https://github.com/zloirock/core-js) from 3.7.0 to 3.8.0.
- [Release notes](https://github.com/zloirock/core-js/releases)
- [Changelog](https://github.com/zloirock/core-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/zloirock/core-js/compare/v3.7.0...v3.8.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump symfony/maker-bundle from 1.23.0 to 1.24.1

Bumps [symfony/maker-bundle](https://github.com/symfony/maker-bundle) from 1.23.0 to 1.24.1.
- [Release notes](https://github.com/symfony/maker-bundle/releases)
- [Changelog](https://github.com/symfony/maker-bundle/blob/main/CHANGELOG.md)
- [Commits](https://github.com/symfony/maker-bundle/compare/v1.23.0...v1.24.1)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump sweetalert2 from 10.10.2 to 10.10.4

Bumps [sweetalert2](https://github.com/sweetalert2/sweetalert2) from 10.10.2 to 10.10.4.
- [Release notes](https://github.com/sweetalert2/sweetalert2/releases)
- [Changelog](https://github.com/sweetalert2/sweetalert2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sweetalert2/sweetalert2/compare/v10.10.2...v10.10.4)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump phpstan/phpstan-doctrine from 0.12.24 to 0.12.25

Bumps [phpstan/phpstan-doctrine](https://github.com/phpstan/phpstan-doctrine) from 0.12.24 to 0.12.25.
- [Release notes](https://github.com/phpstan/phpstan-doctrine/releases)
- [Commits](https://github.com/phpstan/phpstan-doctrine/compare/0.12.24...0.12.25)

Signed-off-by: dependabot[bot] <support@github.com>

dmetzner committed:
Merge pull request #1119 from Catrobat/dependabot/npm_and_yarn/sweetalert2-10.10.4

Bump sweetalert2 from 10.10.2 to 10.10.4

dmetzner committed:
Merge pull request #1118 from Catrobat/dependabot/composer/symfony/maker-bundle-1.24.1

Bump symfony/maker-bundle from 1.23.0 to 1.24.1

dmetzner committed:
Merge pull request #1120 from Catrobat/dependabot/composer/phpstan/phpstan-doctrine-0.12.25

Bump phpstan/phpstan-doctrine from 0.12.24 to 0.12.25

dmetzner committed:
Merge pull request #1117 from Catrobat/dependabot/npm_and_yarn/core-js-3.8.0

Bump core-js from 3.7.0 to 3.8.0

dependabot[bot] committed:
Bump eslint-config-standard from 14.1.1 to 16.0.2

Bumps [eslint-config-standard](https://github.com/standard/eslint-config-standard) from 14.1.1 to 16.0.2.
- [Release notes](https://github.com/standard/eslint-config-standard/releases)
- [Changelog](https://github.com/standard/eslint-config-standard/blob/master/CHANGELOG.md)
- [Commits](https://github.com/standard/eslint-config-standard/compare/v14.1.1...v16.0.2)

Signed-off-by: dependabot[bot] <support@github.com>

dmetzner committed:
Hotfix new ESLINT issues

dmetzner committed:
Merge pull request #1097 from Catrobat/dependabot/npm_and_yarn/eslint-config-standard-16.0.2

Bump eslint-config-standard from 14.1.1 to 16.0.2

dmetzner committed:
Hotfix PHPStan errors

- Bump minor packages
- Fix MetaData import

dmetzner committed:
Merge pull request #1109 from Catrobat/dependabot/composer/vimeo/psalm-4.2.1

Bump vimeo/psalm from 4.1.1 to 4.2.1

dependabot[bot] committed:
Bump webpack-notifier from 1.10.1 to 1.11.0

Bumps [webpack-notifier](https://github.com/Turbo87/webpack-notifier) from 1.10.1 to 1.11.0.
- [Release notes](https://github.com/Turbo87/webpack-notifier/releases)
- [Changelog](https://github.com/Turbo87/webpack-notifier/blob/master/CHANGELOG.md)
- [Commits](https://github.com/Turbo87/webpack-notifier/compare/v1.10.1...v1.11.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump phpstan/phpstan from 0.12.57 to 0.12.58

Bumps [phpstan/phpstan](https://github.com/phpstan/phpstan) from 0.12.57 to 0.12.58.
- [Release notes](https://github.com/phpstan/phpstan/releases)
- [Commits](https://github.com/phpstan/phpstan/compare/0.12.57...0.12.58)

Signed-off-by: dependabot[bot] <support@github.com>

dmetzner committed:
Merge pull request #1124 from Catrobat/dependabot/composer/phpstan/phpstan-0.12.58

Bump phpstan/phpstan from 0.12.57 to 0.12.58

dmetzner committed:
Merge pull request #1123 from Catrobat/dependabot/npm_and_yarn/webpack-notifier-1.11.0

Bump webpack-notifier from 1.10.1 to 1.11.0

dmetzner committed:
Bump wapmorgan/php-deprecation-detector

crowdin-bot committed:
New Crowdin translations by Github Action

dependabot[bot] committed:
Bump lexik/jwt-authentication-bundle from 2.10.1 to 2.10.3

Bumps [lexik/jwt-authentication-bundle](https://github.com/lexik/LexikJWTAuthenticationBundle) from 2.10.1 to 2.10.3.
- [Release notes](https://github.com/lexik/LexikJWTAuthenticationBundle/releases)
- [Changelog](https://github.com/lexik/LexikJWTAuthenticationBundle/blob/master/CHANGELOG.md)
- [Commits](https://github.com/lexik/LexikJWTAuthenticationBundle/compare/v2.10.1...v2.10.3)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump symfony/dotenv from 5.1.9 to 5.2.0

Bumps [symfony/dotenv](https://github.com/symfony/dotenv) from 5.1.9 to 5.2.0.
- [Release notes](https://github.com/symfony/dotenv/releases)
- [Changelog](https://github.com/symfony/dotenv/blob/5.x/CHANGELOG.md)
- [Commits](https://github.com/symfony/dotenv/compare/v5.1.9...v5.2.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump symfony/yaml from 5.1.9 to 5.2.0

Bumps [symfony/yaml](https://github.com/symfony/yaml) from 5.1.9 to 5.2.0.
- [Release notes](https://github.com/symfony/yaml/releases)
- [Changelog](https://github.com/symfony/yaml/blob/5.x/CHANGELOG.md)
- [Commits](https://github.com/symfony/yaml/compare/v5.1.9...v5.2.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump symfony/intl from 5.1.9 to 5.2.0

Bumps [symfony/intl](https://github.com/symfony/intl) from 5.1.9 to 5.2.0.
- [Release notes](https://github.com/symfony/intl/releases)
- [Changelog](https://github.com/symfony/intl/blob/5.x/CHANGELOG.md)
- [Commits](https://github.com/symfony/intl/compare/v5.1.9...v5.2.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump symfony/process from 5.1.9 to 5.2.0

Bumps [symfony/process](https://github.com/symfony/process) from 5.1.9 to 5.2.0.
- [Release notes](https://github.com/symfony/process/releases)
- [Changelog](https://github.com/symfony/process/blob/5.x/CHANGELOG.md)
- [Commits](https://github.com/symfony/process/compare/v5.1.9...v5.2.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump symfony/phpunit-bridge from 5.1.9 to 5.2.0

Bumps [symfony/phpunit-bridge](https://github.com/symfony/phpunit-bridge) from 5.1.9 to 5.2.0.
- [Release notes](https://github.com/symfony/phpunit-bridge/releases)
- [Changelog](https://github.com/symfony/phpunit-bridge/blob/5.x/CHANGELOG.md)
- [Commits](https://github.com/symfony/phpunit-bridge/compare/v5.1.9...v5.2.0)

Signed-off-by: dependabot[bot] <support@github.com>

dmetzner committed:
Fix PhpUnit issues

dmetzner committed:
Merge pull request #1126 from dmetzner/bumpWap

Bump wapmorgan/php-deprecation-detector

dmetzner committed:
Merge pull request #1134 from Catrobat/dependabot/composer/symfony/phpunit-bridge-5.2.0

Bump symfony/phpunit-bridge from 5.1.9 to 5.2.0

dmetzner committed:
Merge pull request #1133 from Catrobat/dependabot/composer/symfony/process-5.2.0

Bump symfony/process from 5.1.9 to 5.2.0

dmetzner committed:
Merge pull request #1131 from Catrobat/dependabot/composer/symfony/intl-5.2.0

Bump symfony/intl from 5.1.9 to 5.2.0

dmetzner committed:
Merge pull request #1130 from Catrobat/dependabot/composer/symfony/yaml-5.2.0

Bump symfony/yaml from 5.1.9 to 5.2.0

dmetzner committed:
Merge pull request #1129 from Catrobat/dependabot/composer/symfony/dotenv-5.2.0

Bump symfony/dotenv from 5.1.9 to 5.2.0

dmetzner committed:
Merge pull request #1128 from Catrobat/dependabot/composer/lexik/jwt-authentication-bundle-2.10.3

Bump lexik/jwt-authentication-bundle from 2.10.1 to 2.10.3

dmetzner committed:
Merge pull request #1127 from Catrobat/l10n_crowdin_action

New Crowdin translations by Github Action

HCrane committed:
fixes

dmetzner committed:
SHARE-410 Minor design fix (termsOfUse)

  - single column on small devices

dmetzner committed:
Merge pull request #1086 from HCrane/SHARE-413

SHARE-413 - HOTFIX API MAIL

HCrane committed:
Merge pull request #1135 from dmetzner/SHARE-410

SHARE-410 Minor design fix (termsOfUse)

dmetzner committed:
Bump capi & deps

dmetzner committed:
Fix medialibrary API to v47

HCrane committed:
fix unit tests

HCrane committed:
Merge pull request #818 from amelak9/SHARE-341/CAPI_Entities_Inconsistency

SHARE-341/CAPI_Entities_Inconsistency

dmetzner committed:
Merge pull request #1115 from Catrobat/dependabot/composer/catrobat/capi-1.0.47

Bump catrobat/capi from 1.0.43 to 1.0.47

dmetzner committed:
Bump CAPI to v48 and fix/adapt PUT user

maxfazeli-dev committed:
SHARE-414 Project visibility gets updated only after confirmation

The AJAX call now gets executed after the user has made a decision in the confirmation dialog.

dmetzner committed:
Merge pull request #761 from JackShahhoud/SHARE-352

SHARE-352 API - PUT /user

dmetzner committed:
Merge pull request #1095 from hungryapeman/SHARE-414

SHARE-414 Project visibility gets updated only after confirmation

dependabot[bot] committed:
Bump sweetalert2 from 10.10.4 to 10.11.0

Bumps [sweetalert2](https://github.com/sweetalert2/sweetalert2) from 10.10.4 to 10.11.0.
- [Release notes](https://github.com/sweetalert2/sweetalert2/releases)
- [Changelog](https://github.com/sweetalert2/sweetalert2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sweetalert2/sweetalert2/compare/v10.10.4...v10.11.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @symfony/webpack-encore from 0.31.0 to 0.31.1

Bumps [@symfony/webpack-encore](https://github.com/symfony/webpack-encore) from 0.31.0 to 0.31.1.
- [Release notes](https://github.com/symfony/webpack-encore/releases)
- [Changelog](https://github.com/symfony/webpack-encore/blob/main/CHANGELOG.md)
- [Commits](https://github.com/symfony/webpack-encore/compare/v0.31.0...v0.31.1)

Signed-off-by: dependabot[bot] <support@github.com>

dmetzner committed:
Merge pull request #1138 from Catrobat/dependabot/npm_and_yarn/symfony/webpack-encore-0.31.1

Bump @symfony/webpack-encore from 0.31.0 to 0.31.1

dmetzner committed:
Merge pull request #1137 from Catrobat/dependabot/npm_and_yarn/sweetalert2-10.11.0

Bump sweetalert2 from 10.10.4 to 10.11.0

dmetzner committed:
SHARE-151 - Part 1: Theme & flavor rework

- Remove liipBundle
- Rework ThemeRequestListener (old FlavorListener)
- Add new Catrobat favicon global for all themes
- Only one title that is consistent accross all themes
- Slight refactoring to define the difference between theme and flavor a little bit clearer
- 'app' as the official umbrella term for themes
- 'pocketcode' as default flavor
- API routes should always return 'app' in the URI
- Added PhpUnit tests for theming/flavoring

dependabot[bot] committed:
Bump sweetalert2 from 10.11.0 to 10.11.1

Bumps [sweetalert2](https://github.com/sweetalert2/sweetalert2) from 10.11.0 to 10.11.1.
- [Release notes](https://github.com/sweetalert2/sweetalert2/releases)
- [Changelog](https://github.com/sweetalert2/sweetalert2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sweetalert2/sweetalert2/compare/v10.11.0...v10.11.1)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @symfony/webpack-encore from 0.31.1 to 0.33.0

Bumps [@symfony/webpack-encore](https://github.com/symfony/webpack-encore) from 0.31.1 to 0.33.0.
- [Release notes](https://github.com/symfony/webpack-encore/releases)
- [Changelog](https://github.com/symfony/webpack-encore/blob/main/CHANGELOG.md)
- [Commits](https://github.com/symfony/webpack-encore/compare/v0.31.1...v0.33.0)

Signed-off-by: dependabot[bot] <support@github.com>

dmetzner committed:
Merge pull request #1140 from Catrobat/dependabot/npm_and_yarn/symfony/webpack-encore-0.33.0

Bump @symfony/webpack-encore from 0.31.1 to 0.33.0

dmetzner committed:
Merge pull request #1139 from Catrobat/dependabot/npm_and_yarn/sweetalert2-10.11.1

Bump sweetalert2 from 10.11.0 to 10.11.1

dependabot[bot] committed:
Bump webpack-notifier from 1.11.0 to 1.11.1

Bumps [webpack-notifier](https://github.com/Turbo87/webpack-notifier) from 1.11.0 to 1.11.1.
- [Release notes](https://github.com/Turbo87/webpack-notifier/releases)
- [Changelog](https://github.com/Turbo87/webpack-notifier/blob/master/CHANGELOG.md)
- [Commits](https://github.com/Turbo87/webpack-notifier/compare/v1.11.0...v1.11.1)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump sass from 1.29.0 to 1.30.0

Bumps [sass](https://github.com/sass/dart-sass) from 1.29.0 to 1.30.0.
- [Release notes](https://github.com/sass/dart-sass/releases)
- [Changelog](https://github.com/sass/dart-sass/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sass/dart-sass/compare/1.29.0...1.30.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump eslint from 7.14.0 to 7.15.0

Bumps [eslint](https://github.com/eslint/eslint) from 7.14.0 to 7.15.0.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/master/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v7.14.0...v7.15.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump sweetalert2 from 10.11.1 to 10.12.0

Bumps [sweetalert2](https://github.com/sweetalert2/sweetalert2) from 10.11.1 to 10.12.0.
- [Release notes](https://github.com/sweetalert2/sweetalert2/releases)
- [Changelog](https://github.com/sweetalert2/sweetalert2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sweetalert2/sweetalert2/compare/v10.11.1...v10.12.0)

Signed-off-by: dependabot[bot] <support@github.com>

dmetzner committed:
Merge pull request #1146 from Catrobat/dependabot/npm_and_yarn/webpack-notifier-1.11.1

Bump webpack-notifier from 1.11.0 to 1.11.1

dmetzner committed:
Merge pull request #1147 from Catrobat/dependabot/npm_and_yarn/sass-1.30.0

Bump sass from 1.29.0 to 1.30.0

dmetzner committed:
Merge pull request #1148 from Catrobat/dependabot/npm_and_yarn/eslint-7.15.0

Bump eslint from 7.14.0 to 7.15.0

dmetzner committed:
Merge pull request #1149 from Catrobat/dependabot/npm_and_yarn/sweetalert2-10.12.0

Bump sweetalert2 from 10.11.1 to 10.12.0

dependabot[bot] committed:
Bump core-js from 3.8.0 to 3.8.1

Bumps [core-js](https://github.com/zloirock/core-js) from 3.8.0 to 3.8.1.
- [Release notes](https://github.com/zloirock/core-js/releases)
- [Changelog](https://github.com/zloirock/core-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/zloirock/core-js/compare/v3.8.0...v3.8.1)

Signed-off-by: dependabot[bot] <support@github.com>

HCrane committed:
Merge pull request #1143 from dmetzner/SHARE-151

SHARE-151 / Part 1 - Theme & flavor rework

dmetzner committed:
Merge pull request #1150 from Catrobat/dependabot/npm_and_yarn/core-js-3.8.1

Bump core-js from 3.8.0 to 3.8.1

mahmoudZkhalil committed:
SHARE-256 Refactoring - Adminarea - Tools | Logs

Separate functionality in methods
Add setter and getter
Write phpUnit tests
Create invoke method for testing private and protected methods

mahmoudZkhalil committed:
SHARE-256 Refactoring - Adminarea - Tools | Logs

Separate functionality in methods
Add setter and getter
Write phpUnit tests
Create invoke method for testing private and protected methods

mahmoudZkhalil committed:
Merge remote-tracking branch 'origin/SHARE-256' into SHARE-256-admin-area-tools-logs

dmetzner committed:
Hotfix: adapt to new xdebug version

HCrane committed:
Merge pull request #1156 from dmetzner/xdebug

Hotfix: adapt to new xdebug version

dependabot[bot] committed:
Bump sweetalert2 from 10.12.0 to 10.12.4

Bumps [sweetalert2](https://github.com/sweetalert2/sweetalert2) from 10.12.0 to 10.12.4.
- [Release notes](https://github.com/sweetalert2/sweetalert2/releases)
- [Changelog](https://github.com/sweetalert2/sweetalert2/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sweetalert2/sweetalert2/compare/v10.12.0...v10.12.4)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump js-yaml from 3.14.0 to 3.14.1

Bumps [js-yaml](https://github.com/nodeca/js-yaml) from 3.14.0 to 3.14.1.
- [Release notes](https://github.com/nodeca/js-yaml/releases)
- [Changelog](https://github.com/nodeca/js-yaml/blob/3.14.1/CHANGELOG.md)
- [Commits](https://github.com/nodeca/js-yaml/compare/3.14.0...3.14.1)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump webpack-notifier from 1.11.1 to 1.12.0

Bumps [webpack-notifier](https://github.com/Turbo87/webpack-notifier) from 1.11.1 to 1.12.0.
- [Release notes](https://github.com/Turbo87/webpack-notifier/releases)
- [Changelog](https://github.com/Turbo87/webpack-notifier/blob/master/CHANGELOG.md)
- [Commits](https://github.com/Turbo87/webpack-notifier/compare/v1.11.1...v1.12.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump actions/setup-node from v2.1.2 to v2.1.3

Bumps [actions/setup-node](https://github.com/actions/setup-node) from v2.1.2 to v2.1.3.
- [Release notes](https://github.com/actions/setup-node/releases)
- [Commits](https://github.com/actions/setup-node/compare/v2.1.2...27082cecf3ff7a1742dbd5e12605f0cb59dce2d9)

Signed-off-by: dependabot[bot] <support@github.com>

dmetzner committed:
Merge pull request #1157 from Catrobat/dependabot/github_actions/actions/setup-node-v2.1.3

Bump actions/setup-node from v2.1.2 to v2.1.3

dmetzner committed:
Merge pull request #1152 from Catrobat/dependabot/npm_and_yarn/webpack-notifier-1.12.0

Bump webpack-notifier from 1.11.1 to 1.12.0

dmetzner committed:
Merge pull request #1153 from Catrobat/dependabot/npm_and_yarn/js-yaml-3.14.1

Bump js-yaml from 3.14.0 to 3.14.1

dmetzner committed:
Merge pull request #1155 from Catrobat/dependabot/npm_and_yarn/sweetalert2-10.12.4

Bump sweetalert2 from 10.12.0 to 10.12.4

dmetzner committed:
SHARE-418 Token for deprecated report API

- deprecated report API support uplaod token
- still support bearer token
- Only add minimum behavior tests since it's deprecated

HCrane committed:
Merge pull request #1158 from dmetzner/SHARE-418

SHARE-418 Token for deprecated report API

dmetzner committed:
SHARE-417 Register API return token

To be finished with refresh tokens..

Misc:
 - Bump capi & minor deps.
 - new code fixes

HCrane committed:
Merge pull request #1159 from dmetzner/SHARE-417

SHARE-417 Register API return token

mahmoudZkhalil committed:
Merge remote-tracking branch 'origin/SHARE-256' into SHARE-256-admin-area-tools-logs

mahmoudZkhalil committed:
SHARE-256
php cs fix

mahmoudZkhalil committed:
SHARE-256
change test method annotation

mahmoudZkhalil committed:
SHARE-256
remove slash frm test method annotation

mahmoudZkhalil committed:
SHARE-256
change test method name

mahmoudZkhalil committed:
SHARE-256
remove covers annotation from test methods
